lizmap/modules/view/controllers/lizMap.classic.php

```diff
@@ -145,8 +145,21 @@ public function index()
         $www = $confUrlEngine['jelixWWWPath'];
         $rep->addJSLink($www.'jquery/include/jquery.include.js');
         $rep->addJSLink($www.'js/jforms_jquery.js');
-        $rep->addJSLink($www.'js/jforms/datepickers/default/init.js');
-        $rep->addJSLink($www.'js/jforms/datepickers/default/ui.en.js');
+
+        // Add datepickers jForms js
+        $confDate = &jApp::config()->datepickers;
+        $rep->addJSLink($confDate['default']);
+        if (isset($confDate['default.js'])) {
+            $js = $confDate['default.js'];
+            foreach($js as $file) {
+                $file = str_replace('$lang', jLocale::getCurrentLang(), $file);
```

Comment:
@laurentj I have found an issue, If the lang is set to `pt_PT`, getCurrentLang() returns `en` and not `pt`. Do you knw where it comes from ?

Reply:
an error in my branches, no issue.